### ISSUE TITLE
More than one file was found with OS independent path

### ISSUE THREAD

**Issue by michalstodolny** [mikepenz/AboutLibraries#439]
## About this issue
Given:
`implementation "com.mikepenz:aboutlibraries:7.0.2"`

When:
`./gradlew :app:transformResourcesWithMergeJavaResForDebug`

Then:
Exception is thrown:
> More than one file was found with OS independent path 'META-INF/library-core_release.kotlin_module'

Also, tried to exclude the file explicitely from packaging but then Material Drawer crashes in runtime: 
> android {
    packagingOptions {
       exclude 'META-INF/library-core_release.kotlin_module'
    }
}

## Details
- Used library version: 7.0.2
- Used support library version: 28.0.0
- Used gradle build tools version: 28.0.3
- Android Studio version: 3.5
- Other used libraries, potential conflicting libraries: `com.mikepenz:materialdrawer:6.1.2`

**Comment by motorro** [comment]
[Here](https://stackoverflow.com/questions/44509608/duplicate-files-copied-in-apk-meta-inf-library-release-kotlin-module) is the potential solution to the issue. The thing is kotlin compiler names it's metadata using the gradle module name. Thus having two libraries that were built from `library` folder in the same project may lead to conflict. I suggest to put something like this to `android` section as the SO answer above suggests:
```groovy
    kotlinOptions {
        freeCompilerArgs += ['-module-name', "mikepenz-aboutlibraries"]
    }
```
Thus the library metadata will be unique.

**Comment by mikepenz** [comment]
@motorro thank you. I will look into this

**Comment by michalstodolny** [comment]
@motorro thank you for the solution proposal. I tried it but without success. Although now the file name is correctly prefixed upon packaging, the issue still persists. I can see it's caused by `AboutLibrary`'s dependencies:

- `com.mikepenz:iconics-core:4.0.0`
- `com.mikepenz:fastadapter:4.0.1`

![kotlin-module-issue](https://user-images.githubusercontent.com/10180716/63726834-b1624800-c85e-11e9-8ab9-4e8cd5025953.png)

**Comment by motorro** [comment]
@michalstodolny, yes. That's the same issue since both libraries you mention have `library-core` modules. I wish kotlin had more sophisticated way to name their metadata by default but AFAIK there is no other way to overcome this other than:
1. Including the above mentioned option in a library project
2. Naming library modules in a `more unique way`

**Comment by mikepenz** [comment]
@michalstodolny @motorro 

I have updated the libs and published them with a more unique (POM-ID) kotlin-module name. so those conflicts should no longer happen.

**Comment by mikepenz** [comment]
this is included in v7.0.3 

Iconics 4.0.1-b02
FastAdapter 4.1.0-b02
(and will do the same for materialdrawer)
